packages/components/src/toggle-group-control/stories/index.js

```diff
@@ -131,26 +130,27 @@ WithAriaLabel.args = {
 	],
 };
 
-export const WithIcons = _default.bind( {} );
-WithIcons.args = {
-	...Default.args,
-	options: [
-		{
-			label: 'Uppercase',
-			value: 'uppercase',
-			icon: formatUppercase,
-		},
-		{
-			label: 'Lowercase',
-			value: 'lowercase',
-			icon: formatLowercase,
-		},
-		{
-			label: 'Capitalize',
-			value: 'capitalize',
-			icon: formatCapitalize,
-		},
-	],
+export const WithIcons = () => {
+	const [ state, setState ] = useState();
+	return (
+		<ToggleGroupControl
+			onChange={ setState }
+			value={ state }
+			label={ 'With icons' }
+			hideLabelFromVision
+		>
+			<ToggleGroupControlOptionIcon
+				value="uppercase"
+				icon={ formatUppercase }
+				showTooltip={ true }
+			/>
+			<ToggleGroupControlOptionIcon
+				value="lowercase"
+				icon={ formatLowercase }
+				showTooltip={ true }
+			/>
+		</ToggleGroupControl>
```

Comment:
Since `showTooltip` is `true`, let's add an `aria-label` prop (otherwise the tooltip won't be shown)

 ```suggestion
			<ToggleGroupControlOptionIcon
				value="uppercase"
				icon={ formatUppercase }
				showTooltip={ true }
				aria-label="Uppercase"
			/>
			<ToggleGroupControlOptionIcon
				value="lowercase"
				icon={ formatLowercase }
				showTooltip={ true }
				aria-label="Lowercase"
			/>
		</ToggleGroupControl>
```